website/docs/d/tm_supervisor.html.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_tm_supervisor"
+sidebar_current: "docs-vcd-datasource-tm-supervisor"
+description: |-
+  Provides a data source to read Supervisors in VMware Cloud Foundation Tenant Manager.
+---
+
+# vcd\_tm\_supervisor
+
+Provides a data source to read Supervisors in VMware Cloud Foundation Tenant Manager.
+
+## Example Usage
+
+```hcl
+data "vcd_tm_supervisor" "one" {
+  name = "my-supervisor-name"
+
+  depends_on = [vcd_vcenter.one]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of Supervisor.
```

Comment:
Very similar question to the one I had about storage policies. Aren't we missing vcenter_id here?

Also, as it stands right now, you need to have depends_on vCenter clause to make it work in a fresh setup:

```
data "vcd_tm_supervisor" "test" {
  name = var.supervisor_name

  depends_on = [vcd_vcenter.test]
}
```

Reply:
This is now patched to require `vcenter_id`